recipes/1-Insights/conditioned-insights.ipynb

```diff
@@ -0,0 +1,309 @@
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "# Section 1: Load, Train, Analyze\n",
+    "\n",
+    "The [basic workflow guide](https://docs.howso.com/user_guide/basics/basic_workflow.html) goes into more specifics about the individual details of this section. This recipe will focus more on the insights."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = fetch_data('adult', local_cache_dir=\"../../data/adult\")\n",
+    "\n",
+    "train_data = df.iloc[:-30]\n",
+    "new_data = df[~df.index.isin(train_data.index)]\n",
+    "features = infer_feature_attributes(train_data)\n",
+    "\n",
+    "# Subsample the data to ensure the example runs quickly\n",
+    "df = df.sample(2000)\n",
+    "\n",
+    "# Split out the last row for a prediction set and drop the Action Feature\n",
+    "test_case = df.iloc[[-1]].copy()\n",
+    "\n",
+    "# Set the sex to always be female\n",
+    "test_case.at[0, 'sex'] = 0\n",
+    "\n",
+    "df.drop(df.index[-1], inplace=True)\n",
+    "test_case = test_case.drop('target', axis=1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "The following parameters from configuration file will override the Amalgam parameters set in the code: {'trace', 'library_path'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "t = Trainee(features=features)\n",
+    "\n",
+    "action_features = ['target']\n",
+    "context_features = features.get_names(without=action_features)\n",
+    "\n",
+    "t.train(df)\n",
+    "\n",
+    "t.analyze(context_features=context_features, action_features=action_features)\n",
+    "\n",
+    "t.react_into_trainee(action_feature=action_features[0], residuals=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prediction Stats\n",
+    "\n",
+    "Howso provides a variety of prediction stats based on the variable type, whether it is continuous or nominal. These prediction stats gives insight into the predictive performance of the Trainee as well as insights into the data itself. \n",
+    "\n",
+    "The [bias_mitigation.ipynb](https://github.com/howsoai/howso-engine-recipes/blob/main/recipes/2-Workflows/bias_mitigation/bias_mitigation.ipynb) recipe highlights looking for bias in the data through the analysis of individual features. This recipe demonstrates how insights like overall data bias can be explored."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Global Prediction Stats\n",
+    "\n",
+    "\n",
+    "Global prediction stats provides an overall view on how accurate the Trainee is. This can be a great introductory look into the usability of the data and Trainee. Most machine learning models are evaluated on similiar global metrics.\n",
```

Comment:
Can we give users an example of what the global prediction stats are to root them in what we are talking about: accuracy, precision, ...